packages/geo/src/index.ts

```diff
@@ -4,3 +4,4 @@ export { GeoJson } from './geo.json';
 export { Epsg, EpsgCode } from './epsg';
 export { TileMatrixSet, Tile } from './tile.matrix.set';
 export { WmtsLayer, WmtsProvider } from './wmts/wmts';
+export { TileSetName, TileSetNameValues } from './tile.set.name';
```

Comment:
This is also quite specific to our process, is there another place this could live?

maybe something like `shared` ?